src/asmcnc/comms/router_machine.py

```diff
@@ -1897,6 +1900,11 @@ def reset_tuning_flags(self):
         self.temp_toff = 2
         self.temp_sgt = 0
 
+    def motor_driver_temp_in_range(self, temp_to_assess):
+
+        if (self.lower_temp_limit <= temp_to_assess <= self.upper_temp_limit): return True
+        else: return False
+
```

Comment:
function could be simplified:

`def motor_driver_temp_in_range(self, temp_to_assess):
    return True if self.lower_temp_limit <= temp_to_assess <= self.upper_temp_limit else False`

Reply:
This guy's a genius

Reply:
good catch, corrected

Reply:
Also as before you had it like 
if ___: return True 
else: return False 

you could do 

return True if ___ else return False 

which is a bit cleaner